cx_Freeze/freezer.py

```diff
@@ -749,7 +757,7 @@ def _WriteModules(self, fileName, finder):
 
     def Freeze(self):
         self.finder = None
-        self.excludeModules = {}
+        self.excludeModules = self.excludes
```

Comment:
Sorry for the delay. I think you found an issue that I already suspected, but not checked it yet!
But, here, if you initialize excludeModules with excludes, only the user excludes will be excluded. There are excludes from hooks (See hooks.py lines with ExcludeModule).
I think that the correct is use self.finder.excludes, so all excludes will be excluded.

Reply:
You're very much right, thanks! I've changed it to `self.finder.excludes`

Reply:
There is a conflict. Can you resolve it? And, I think you can use it directly in the WriteModules and delete this self.excludeModules. It is used only for this.

Reply:
Done :)